app/Nova/Address.php

```diff
@@ -45,6 +46,8 @@ public function fields(Request $request)
                 Text::make('Postal Code'),
                 Text::make('Country'),
             ]),
+
+            BelongsTo::make('user', 'user', User::class),
```

Comment:
Should be optional. Otherwise might be best to create new resources and models to demonstrate the usage.